enclave/core/oe_alloc_thread.h

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#ifdef __cplusplus
```

Comment:
You can use `OE_EXTERNC_BEGIN` and `OE_EXTERNC_END` macros instead

Reply:
Ah, I did not know about those, will do.